core/src/main/resources/defaults/blueprints/hdp30-ds-standalone.bp

```diff
@@ -0,0 +1,206 @@
+{
+  "description": "Useful for data science with Spark and Zeppelin",
+  "blueprint": {
+    "Blueprints": {
+      "blueprint_name": "hdp30-ds-standalone-v1",
+      "stack_name": "HDP",
+      "stack_version": "3.0"
+    },
+    "configurations": [
+      {
+        "yarn-site": {
+          "properties_attributes": {},
+          "properties": {
+            "hadoop.registry.dns.enabled": "false",
+            "yarn.log-aggregation.fs-support-append": "false",
+            "yarn.log-aggregation.file-formats": "TFile",
+            "yarn.timeline-service.versions" : "2.0f",
+            "yarn.resourcemanager.cluster-id" : "{{{ general.uuid }}}",
+            "yarn.log-aggregation-status.time-out.ms": "1200000"
+          }
+        }
+      },
+      {
+        "zeppelin-site": {
+          "properties_attributes": {},
+          "properties": {
+            "zeppelin.notebook.dir": "s3a://ddas-warehouse/spark/zeppelin/notebook",
```

Comment:
Probably should be removed (populated by https://github.com/hortonworks/cloudbreak/blob/master/template-manager-blueprint/src/main/resources/definitions/cloud-storage-location-specification.json).